packages/utils/src/types/actions.ts

```diff
@@ -292,6 +294,26 @@ export const CreateActionDetailSchema = z.object({
 export type CreateActionForm = z.infer<typeof CreateActionFormSchema>
 export type CreateActionDetail = z.infer<typeof CreateActionDetailSchema>
 
+/*
+COMPLETE
+*/
+export type CompleteActionParams = {
+  boostId?: string
```

Comment:
I think we will also want to take in the chainId as an actionParam.

Reply:
Im not sure if we can make it work for all networks at the same time?

Reply:
I added `chainId` as optional. It should allow us to enable for all networks. Will update if we need to enforce a `chainId`